bin/pry

```diff
@@ -52,4 +52,47 @@ if Config.development?
   Pry.prepend(PryReloader)
 end
 
+if Config.production? || ENV["INGEST_KEY"]
+  PRY_LOGGER = LogDnaBatcher.new(ENV["INGEST_KEY"])
```

Comment:
If ingest key is not set in production, I think we should print an error message and exit.